### DISCUSSION TITLE
skipping due to mismatch

### DISCUSSION THREAD

**Discussion by aleJohnny** [mxpv/podsync#741] [General]
Hi,

Just setup **podsync** in a docker container, but every channel I add to the config.toml, returns "skipping due to mismatch episode_id=XXXXXX filter=title" in podsync.log.

I searched for other issues here, but didn't found a solution... Am I missing something?

Thanks.

**Comment by manumora** [comment]
I have two feeds added, one channel and one playlist. Channel downloads work fine, but the second feed, the playlist fails. Each time I add a new video to the playlist it is properly detected but it says "skipping due to mismatch"

**Comment by manumora** [comment]
@aleJohnny I have found the issue. That happens because you have added to config.toml the following line:
`filters = { title = "whatever thing" }`
Remove it and will be fine.
